src/include/rccl_float8.h

```diff
@@ -1016,6 +1059,6 @@ inline __host__ __device__ T explicit_downcast(Ta a, uint32_t rng)
 
 // =================================================================================================
 
-#endif // __cplusplus < 201103L || (!defined(__HCC__) && !defined(__HIPCC__))
+#endif
```

Comment:
Let's ditch this and change it if we change the logic above.
```suggestion
#endif // __cplusplus < 201103L || (!defined(__HCC__) && !defined(__HIPCC__))
```